azure/storage-account-sample/pom.xml

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>com.yanchware.fractal</groupId>
+    <artifactId>storage-account.sample</artifactId>
```

Comment:
Is is correct to have also `-` and `.` here?

Reply:
Thanks for having spotted it out!
I'll fix it